### PR TITLE
CI improvements

### PR DESCRIPTION
**Pull request by MahdiBM**

### CODE REVIEW

**Review by 0xTim — APPROVED** [review]

**Comment by MahdiBM** [comment]
@gwynne do you approve now? 😛
I removed the wait-for-tests stuff.
Also @0xTim was not too happy with it either so i guess he should like it too.

File: .github/workflows/deploy-penny.yml [hunk 1]
```diff
@@ -33,20 +38,23 @@ jobs:
           # be deployed to ECS.
           docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -f Dockerfile .
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
-          echo "image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >>"${GITHUB_OUTPUT}"
+          echo "image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >> $GITHUB_OUTPUT
```
Comment (gwynne) [inline .github/workflows/deploy-penny.yml:41]: Why?

**Review by gwynne — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-penny.yml (hunk 1 above)
Comment (MahdiBM) [reply .github/workflows/deploy-penny.yml:41]: Probably my formatter? Not exactly sure

Comment (gwynne) [reply .github/workflows/deploy-penny.yml:41]: Quoting the variable protects from spaces in the variable value. Adding the braces is syntactically purely stylistic in this context, but I prefer it just the same (it's "quoting" for variable names, so you could disambiguate, for example, `"$GITHUB_OUTPUT_FILE"` from `"${GITHUB_OUTPUT}_FILE"` - contrived but you get the idea).

Comment (MahdiBM) [reply .github/workflows/deploy-penny.yml:41]: hmm i use it this way everywhere apparently so i guess i'll keep it this way.

**Pull request merged**